pkg/generator/caches.go

```diff
@@ -254,18 +274,30 @@ func generateListenersAndRouteConfigs(
 
 			externalHTTPSEnvoyListener.FilterChains = append(externalHTTPSEnvoyListener.FilterChains,
 				externalHTTPSEnvoyListenerWithOneCertFilterChain)
+
```

Comment:
Could you drop unnecessary blank line? :sweat_smile: Not only this one, but also there are some unnecessary break lines.